cl/corpus_importer/tasks.py

```diff
@@ -1160,6 +1168,8 @@ def make_docket_by_iquery(
     :param tag_names: A list of strings that should be added to the docket as
     tags
     :param log_results_redis: Log results in redis for the ready mix project
+    :param from_iquery_scrape: Weather this method was invoked by the iquery
```

Comment:
As I said before, I think this would be better as "trigger_signal" or something like that?

Reply:
Sure, changed as described in a previous comment.